.github/workflows/build.yml

```diff
@@ -1,12 +1,21 @@
 name: Build
 
 on:
+  # Run on pushes to master and pushed tags, and on pull requests against master, but ignore the docs folder
   push:
     branches: [ master ]
     tags:
       - 'v*'
+    paths:
```

Comment:
is the intent here just to ignore the docs path and include everything else?

Reply:
Yeh

Reply:
paths-ignore is reccomended here then 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore